src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,563 @@
+been working on adding more support to our lab tooling for animation detection,
+as well making steady improvements to the rendering pipeline diagnostics within
+Chromium.
+
+We'd like to share some recent progress, offer concrete tooling guidance,
+and discuss ideas for future animation smoothness metrics. As always, we
+would love to hear your [feedback](#feedback).
+
+This post will cover three main topics:
+
+- A quick look at animations and animation frames.
+- Our current thoughts on measuring overall animation smoothness.
+- A few practical suggestions for you to leverage in lab tooling today.
+
+## What are animations?
+
+Animations bring content to life! By making content move, especially in
+response to user interactions, animations can make an experience feel more
+natural, understandable, and fun.
+
+But poorly implemented animations, or just adding too many animations, can
+degrade the experience and make it decidedly not fun at all. We've probably all
+interacted with an interface which just added too many "helpful" transition
+effects, which actually become hostile to experience when they perform poorly.
+
+You can read plenty more about [Animations](/animations/), including how to
+[create performant animations](/animations-guide/) and how to [inspect
+animations using developer
+tooling](https://developer.chrome.com/docs/devtools/css/animations/).
+
+### How do animations work?
+
+As a quick recap, the [rendering pipeline](/animations-overview/#pipeline)
+consists of a few, **sequential** stages:
+
+1. <strong style="font-weight:700;color:#6251A2">Style:</strong> Calculate the
+   styles that apply to the elements.
+2. <strong style="font-weight:700;color:#6251A2">Layout:</strong> Generate the
+   geometry and position for each element.
+3. <strong style="font-weight:700;color:#78A55A">Paint:</strong> Fill out the
+   pixels for each element into layers.
+4. <strong style="font-weight:700;color:#78A55A">Composite:</strong> Draw the
+   layers to the screen.
+
+While there are many ways to define animations, they all fundamentally work via
+one of the following:
+
+- Adjusting <strong style="font-weight:700;color:#6251A2">layout</strong>
+   properties.
+- Adjusting <strong style="font-weight:700;color:#78A55A">paint</strong>
+   properties.
+- Adjusting <strong style="font-weight:700;color:#78A55A">composite</strong>
+   properties.
+
+Because these stages are sequential, it is important to define animations in
+terms of properties further down this pipeline. The earlier the update, the
+greater the costs and the less likely it is to be smooth. (See [rendering
+performance](https://developers.google.com/web/fundamentals/performance/rendering)
+for more details.)
+
+So, while it can sometimes be convenient to animate layout properties, there are
+costs to doing so—even if not immediately apparent. Animations should, wherever
+possible, be defined in terms of composite property changes.
```

Comment:
```suggestion
While it can be convenient to animate layout properties, there are
costs to doing so, even if those costs aren't immediately apparent. Animations should be defined in terms of composite property changes wherever possible.
```